interfaces/inc/c_pal/malloc_multi_flex.h

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+#ifndef MALLOC_MULTI_FLEX_H
+#define MALLOC_MULTI_FLEX_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#include <cstdint>
+#else
+#include <stddef.h>
+#include <stdint.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    #define ARG_OVERFLOW_CHECK(arg) \
```

Comment:
>ARG_OVERFLOW_CHECK [](http://example.com/codeflow?start=12&length=18)

we should probably prefix names with something so they aren't likely to collide with other defines that may be included together. E.g. `MALLOC_MULTI_FLEX_ARG_OVERFLOW_CHECK`

Reply:
here and everywhere for the "internal" things